### PR TITLE
fix: Switch to endo version of packages

### LINKED ISSUE #100: Adopt `@endo` packages
Packages like `eventual-send` have moved onto Endo, which means `agoric install` no longer links them from the `agoric-sdk` repo and we end up pulling old versions from npm. These packages should be stable and backwards compatible enough that using their new `@endo` version should work. Probably should double check running the loadgen on old agoric-sdk revisions.

### PR DESCRIPTION
**Pull request by mhofman**
Fixes #100 

Stop using the old `@agoric` dependencies that have since moved to Endo.

Manually tested against `release-pismo`. I believe behavior of packages has stayed backwards compatible but I can verify against older revisions of `agoric-sdk` if needed.

### CODE REVIEW

File: runner/jsconfig.json [hunk 1]
```diff
@@ -25,5 +26,5 @@
     "noUnusedLocals": true
     // "noUnusedParameters": true
   },
-  "include": ["lib/**/*.js", "lib/**/*.d.ts"]
+  "include": ["lib/**/*.js", "lib/**/*.d.ts", "../node_modules/ses/index.d.ts"]
```
Comment (mhofman) [inline runner/jsconfig.json:29]: Not sure why `@endo/init` doesn't entrain the SES globals.

File: loadgen/contract/agent-create-vault.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // @ts-check
 
-import { E } from '@agoric/eventual-send';
-import { Far } from '@agoric/marshal';
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/far';
```
Comment (erights) [inline loadgen/contract/agent-create-vault.js:4]:
```suggestion
import { E, Far } from '@endo/far';
```

File: loadgen/contract/agent-tap-faucet.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 // @ts-check
 
 import { AmountMath } from '@agoric/ertp';
-import { E } from '@agoric/eventual-send';
-import { Far } from '@agoric/marshal';
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/far';
```
Comment (erights) [inline loadgen/contract/agent-tap-faucet.js:5]:
```suggestion
import { E, Far } from '@endo/far';
```

**Review by erights — COMMENTED** [review]
How overlap is there between this PR and https://github.com/Agoric/agoric-sdk/pull/6792 ?

**Comment by mhofman** [comment]
> How overlap is there between this PR and [Agoric/agoric-sdk#6792](https://github.com/Agoric/agoric-sdk/pull/6792) ?

Different repo, so not any?

Is there a published version of `@endo/far` which re-exports `E` already ? I don't think it hurts to explicitly import from `@endo/eventual-send`, right?

**Comment by erights** [comment]
> Different repo, so not any?

Doh!

> Is there a published version of `@endo/far` which re-exports `E` already ? I don't think it hurts to explicitly import from `@endo/eventual-send`, right?

It does not hurt.

**Review by erights — APPROVED** [review]
LGTM

**Pull request merged**